staging/src/k8s.io/apiserver/pkg/server/options/admission.go

```diff
@@ -129,7 +129,23 @@ func (a *AdmissionOptions) ApplyTo(
 }
 
 func (a *AdmissionOptions) Validate() []error {
+	if a == nil {
+		return nil
+	}
+
 	errs := []error{}
+
+	registeredPluginsMap := make(map[string]bool)
```

Comment:
use the string set https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apimachinery/pkg/util/sets/string.go#L30

Reply:
ok